src/unity/python/turicreate/toolkits/recommender/util.py

```diff
@@ -1312,9 +1293,7 @@ def check_type(arg, arg_name, required_type, allowed_types):
                }
 
         with QuietProgress(verbose):
-            response = _turicreate.extensions._recsys.recommend(opt)
-
-        recs = response['data']
+            recs = self.__proxy__.recommend(users, exclude, items, new_observation_data, new_user_data, new_item_data, exclude_known, k, diversity, random_seed)
```

Comment:
This is just a matter of code style, not super important, and we're not very strict about this throughout our codebase -- I think we should maintain the 80 character per line limit as much as possible, but I'll let @znation , @nickjong , and @hoytak chime in here.

Reply:
I didn't see this before my previous comment. [Here is what PEP8 say about it](http://docs.bigchaindb.com/projects/contributing/en/latest/cross-project-policies/python-style-guide.html). I don't think we need to be super strict by keeping it under 100 characters is a good idea.

Reply:
Thanks everyone